### PR TITLE
MANTA-4428 rust-sharkspotter should put each object on a newline in default output file

### PR DESCRIPTION
**Pull request by rjloura**
Also:
MANTA-4492 pin rust-sharkspotter to rust-libmanta 0.1.0
and Tag: v0.1.0

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -2,8 +2,7 @@
 
 extern crate sharkspotter;
```
Comment (bowrocker) [inline src/main.rs]: I don't think you need this anymore with 2018.

File: src/lib.rs [hunk 2]
```diff
@@ -99,7 +99,6 @@ where
     );
 
     match mclient.sql(query.as_str(), vec![], r#"{"timeout": 10000}"#, |a| {
```
Comment (bowrocker) [inline src/lib.rs]: Not really part of this change, but maybe consider moving the SQL statement into its own function, as here:

https://github.com/joyent/boray/blob/master/src/object/update.rs#L108

Or at least extract it from the method. I think it makes it less easier to miss, and update if necessary.

File: src/lib.rs [hunk 3]
```diff
@@ -121,7 +120,7 @@ fn iter_ids<F>(
 where
     F: FnMut(MantaObject, u32) -> Result<(), Error>,
 {
-    let mut mclient = MorayClient::from_str(moray_socket.as_str(), log, None).unwrap();
+    let mut mclient = MorayClient::from_str(moray_socket, log, None).unwrap();
```
Comment (bowrocker) [inline src/lib.rs]: What happens if from_str panics here? Should we handle it?

File: Cargo.toml [hunk 4]
```diff
@@ -9,7 +9,7 @@ assert_cli = "0.6.0"
 serde = { version = "1.0.89", features = ["derive"] }
 serde_json = "1.0.39"
 clap = "2.33.0"
-libmanta = { git = "https://github.com/joyent/rust-libmanta" }
+libmanta = { git = "https://github.com/joyent/rust-libmanta", tag = "v0.1.0" }
```
Comment (bowrocker) [inline Cargo.toml]: 👍

File: src/lib.rs (hunk 3 above)
Comment (rjloura) [reply src/lib.rs]: I changed this one, but this really needs to be fixed throughout.  I filed MANTA-4494.

File: src/lib.rs (hunk 2 above)
Comment (rjloura) [reply src/lib.rs]: Made some changes here.

**Comment by rjloura** [comment]
I pushed this change to gerrit, lets move the conversation over there.
